src/assets/css/sass/_gogo.style.scss

```diff
@@ -8831,3 +8831,27 @@ footer.page-footer {
     }
   }
 }
+
+.OrderBook table {

```

Comment:
do order book more compact, not in all page width with so big rows. Because it should be near the chart.